django/thunderstore/social/api/experimental/views/current_user.py

```diff
@@ -71,7 +77,22 @@ def get_user_profile(user: UserType) -> UserProfile:
     return {
         "username": username,
         "capabilities": capabilities,
-        "has_subscription": has_subscription,
+        "subscription": get_subscription_status(user),
         "rated_packages": rated_packages,
         "teams": teams,
     }
+
+
+def get_subscription_status(user: Optional[UserType]) -> SubscriptionStatus:
+    """
+    Return information regarding user's paid subscription plan.
+
+    TODO: This is just a stub. The real thing (as well as the definition
+    for SubscriptionStatus) should be implemented in our proprietary
+    django_paypal package or some other more suitable location.
+    """
+    if not user:
+        return {"expires": None}
+
+    # TODO: return expiration date if user has active subscription.
+    return {"expires": None}
```

Comment:
I feel it'd make the most sense to allow `null` to represent a missing object, so this should probably return `None` for when a subscription isn't found. Otherwise we'll have to stub all the fields if we add some in the future.

Reply:
This was discussed a bit more in Discord. Decided to leave it as-is for now. Let's change it when we have a concrete reason to. Resolving the comment.